apa.csl

```diff
@@ -129,7 +129,7 @@
       <else>
         <choose>
           <if variable="DOI">
-            <text variable="DOI" prefix="doi:"/>
+            <text variable="DOI" prefix="http://dx.doi.org/"/>
```

Comment:
I know they say dx.doi.org, but could we go with doi.org? It's both shorter and preferred by CrossRef